_inc/client/components/progress-bar/style.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../scss/variables/colors';
+
+.progress-bar {
+	width: 100%;
```

Comment:
Currently the bar covers ~75% of the section width on complete indexing. Would it be a good idea to scale it to span the length of the space?